.gitignore

```diff
@@ -2,7 +2,7 @@
 
 *.exe
 src/bitcoin
-src/bitcoind
+src/bitcoinabc
```

Comment:
My bad. I assumed it was directory since it was listed under 'src'.  Seems misleading as any binaries should be under 'build', or at least that's the assumption I would make.